### PR TITLE
remove mixpanel tracking code

### LINKED ISSUE #5408: Remove Mixpanel tracking code
- [x] bump gemfile
- [x] remove codebase
- [ ] remove config

### PR DESCRIPTION
**Pull request by matallo**
close #5408

REVIEWER: @xavijam
cc @CartoDB/frontend

### CODE REVIEW

**Comment by javisantana** [comment]
does the gemfile.lock to be updated?

**Comment by matallo** [comment]
it doesn't need to (as in we will have a gem that we won't be using), prefer to restore it and bump it in other PR?

File: NEWS.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 * Fixed maps disappearing after creation + navigation to dashboard [#5264](https://github.com/CartoDB/cartodb/issues/5264)
 * Log.append now allows to disable truncating (by default active)
 * Detection of lat/long columns now is done in `ogr2ogr2` rather than rails code [#5349](https://github.com/CartoDB/cartodb/pull/5349)
+* Removed Mixpanel tracking code
```
Comment (xavijam) [inline NEWS.md]: Refering ticket?

**Comment by xavijam** [comment]
Remember to bump the assets version.

**Comment by xavijam** [comment]
I tested in staging (mainly public part) and seems to be :+1:.

**Pull request merged**